ui/src/features/annotator/annotations/annotation-shape.component.test.tsx

```diff
@@ -4,17 +4,25 @@
 import { render, screen } from '@testing-library/react';
 import { describe, expect, it } from 'vitest';
 
-import { getMockedAnnotation } from '../../../tests/test-utils/mocked-annotation';
-import { AnnotationShape } from './annotation-shape';
-import { Annotation, Polygon, Rect } from './types';
+import { getMockedAnnotation } from '../../../../tests/test-utils/mocked-annotation';
+import { response } from '../../dataset/mock-response';
+import { AnnotatorProvider } from '../annotator-provider.component';
+import { Annotation, Polygon, Rect } from '../types';
+import { AnnotationShape } from './annotation-shape.component';
 
 type AnnotationRect = Annotation & { shape: Rect };
 type AnnotationPolygon = Annotation & { shape: Polygon };
 
+const mockMediaItem = response.items[0];
+
 describe('AnnotationShape', () => {
     it('bounding box as a rect', () => {
         const annotation = getMockedAnnotation() as AnnotationRect;
-        render(<AnnotationShape annotation={annotation} />);
+        render(
+            <AnnotatorProvider mediaItem={mockMediaItem}>
```

Comment:
np: `AnnotationShape` does not rely on `AnnotatorProvider` anymore, so it can be removed here

Reply:
Updated